packages/web/config.ts

```diff
@@ -23,10 +23,11 @@ export const CONFIG = {
     'https://ceramic-clay.3boxlabs.com', // testnet
   ceramicNetwork:
     process.env.NEXT_PUBLIC_CERAMIC_NETWORK || 'mainnet' || 'testnet-clay',
-  calendarId: 'nih59ktgafmm64ed4qk6ue8vv4@group.calendar.google.com',
+  calendarId: 'nih59ktgafmm64ed4qk6ue8vv4',
   googleDataAPIKey: process.env.NEXT_PUBLIC_YOUTUBE_API_KEY,
   web3StorageToken: process.env.WEB3_STORAGE_TOKEN,
   openseaAPIKey: process.env.OPENSEA_API_KEY,
   alchemyAPIKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY,
   mainnetRPC: process.env.NEXT_PUBLIC_MAINNET_RPC || 'https://eth.llamarpc.com',
+  metagameCalendarBackend: 'http://localhost:3000/api/events',
```

Comment:
I don't think this will work ;)

Reply:
You can reference `NEXT_PUBLIC_FRONTEND_URL` above to get the current host

Reply:
Done.